### PR TITLE
added Footer in Records in a full-width style

### PR DESCRIPTION
**Pull request by hoseinmirian**

### CODE REVIEW

**Comment by hoseinmirian** [comment]
please note that this will put Footer below all content to be able showing Footer full-width. Ofcourse its not showing Footer fully due to upcoming branchs will fix this issue.

**Review by knirirr — APPROVED** [review]
Presumably this is related to the previous PR on which I commented; it can be hard to work out what's happening unless there's a comment explaining it, other than the title.

File: src/views/Records/Records.vue [hunk 1]
```diff
@@ -72,10 +71,11 @@ import JumpToTop from "@/components/Navigation/jumpToTop";
 import recordsLabels from "@/data/recordsTypes.json"
 import FilterChips from "@/components/Records/Search/Header/FilterChips";
 import filterChipsUtils from "@/utils/filterChipsUtils";
+import Footer from "@/components/Navigation/Footer";
 
 export default {
   name: "Records",
-  components: { JumpToTop, SearchOutput, SearchInput, FilterChips},
+  components: {Footer, JumpToTop, SearchOutput, SearchInput, FilterChips},
```
Comment (terazus) [inline src/views/Records/Records.vue]: Why not handle the footer from App.vue by removing the v-if in the footer?

**Review by terazus — CHANGES_REQUESTED** [review]

File: src/views/Records/Records.vue (hunk 1 above)
Comment (hoseinmirian) [reply src/views/Records/Records.vue]: You are right we can do it. its been done in the next branches. so this will be overwritten..

**Pull request merged**